packages/sanity/src/core/store/_legacy/history/history/Timeline.ts

```diff
@@ -224,7 +228,9 @@ export class Timeline {
       return this.reachedEarliestEntry ? 'invalid' : 'loading'
     }
 
-    const [timestampStr, chunkId] = id.split('/', 3)
+    const idParts = id.split('/', 3)
```

Comment:
Out of curiosity, why the change?

Reply:
Good catch. This is a weird one. Indeed there was nothing wrong with this code. But without this change, it made `@microsoft/api-extractor` trip (unsupported language feature). Will add a comment in the code as well.